pyrightconfig.json

```diff
@@ -5,6 +5,7 @@
     "times_excel_reader.py"
   ],
   "exclude": [
+    "utils/get_times_order.py",
```

Comment:
The latest commit deletes `get_times_order.py`. Should we remove it from here as well?